client/db/src/parity_db.rs

```diff
@@ -39,14 +39,14 @@ pub fn open<H: Clone + AsRef<[u8]>>(
 	db_type: DatabaseType,
 ) -> parity_db::Result<std::sync::Arc<dyn Database<H>>> {
 	let mut config = parity_db::Options::with_columns(path, NUM_COLUMNS as u8);
-	config.sync = true; // Flush each commit
 	if db_type == DatabaseType::Full {
 		let mut state_col = &mut config.columns[columns::STATE as usize];
 		state_col.ref_counted = true;
 		state_col.preimage = true;
 		state_col.uniform = true;
```

Comment:
Since this PR switches to the new database version and breaks compatibility we'd want to finalize new column settings as well. Basically this means enabling compression for STATE, HEADER, BODY, TRANSACTION and JUSTIFICATION columns here

Reply:
`Snappy` or `Lz4`? 

is it enough for me to set

```rust
stat_col.compress = CompressionType::Snappy;
```

on the columns with corresponding indexes?

Reply:
`Lz4` is preferrable. 
Yes, something like `config.columns[columns::BODY as usize] = CompressionType::Lz4;`. Same for the light client case, except column indexes are defined in `crate::light` and there are fewer columns.